src/main/java/com/iota/iri/controllers/ApproveeViewModel.java

```diff
@@ -26,14 +28,20 @@ public ApproveeViewModel(Hash hash) {
         this.hash = hash;
     }
 
+    public ApproveeViewModel(ApproveeViewModel approveeViewModel) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Public method and constructor comments are required](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4733428)